app/models/concerns/orchestration/common.rb

```diff
@@ -5,4 +5,9 @@ def handle_validation_errors
     logger.debug "Error occured during validations of #{self.class.name}: #{e.message}"
     nil
   end
+
+  # override sql logger for all orchestration mix-ins
+  def logger
```

Comment:
Note that `Orchestration::Common` is included only to `DHCP` and `DNS` module. It it actually was included to all models that use orchestration (such as Host::Managed in https://github.com/theforeman/foreman/blob/develop/app/models/host/managed.rb#L53) would use the app logger, while others would use the `sql`. 

Maybe what we need is just to override the logger in all our models to use the `app` logger: it seems ActiveRecord doesn't use this method internally.